src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <stars class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </stars>
+    <stars class="stars stars--1">
```

Comment:
```suggestion
    </stars>

    <stars class="stars stars--1">
```
Good comment from the AI-mentor
Apply it (that's a general rule - take it into account in all tasks)